chart/templates/ingress.yaml

```diff
@@ -55,6 +55,7 @@ spec:
           {{- end }}
         {{- if or (.Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress") (not (.Capabilities.APIVersions.Has "networking.k8s.io/v1beta1/Ingress")) }}
         pathType: ImplementationSpecific
+        path: "/"
         {{- end }}
 {{- if eq .Values.tls "ingress" }}
   tls:
```

Comment:
We have a hard requirement for all issues now that they need some type of tests, could you add unit or integration tests here.

Reply:
I added a test for the helm chart. The change will also automatically be tested with this integration test: https://github.com/rancher/rancher/blob/release/v2.7/tests/validation/tests/v3_api/test_create_ha.py.